### ISSUE TITLE
Check in Activity crashes if open and closed rapidly

### ISSUE THREAD

**Issue by thunter1000** [Flaredown/FlaredownAndroid#44]
If the check in activity is closed before the popular tags has loaded the app crashes...

```
06-15 21:11:14.678 23532-23532/com.flaredown.flaredownApp.DEBUG E/AndroidRuntime: FATAL EXCEPTION: main
  Process: com.flaredown.flaredownApp.DEBUG, PID: 23532
  java.lang.NullPointerException: Attempt to invoke virtual method 'android.content.res.Resources android.content.Context.getResources()' on a null object reference
      at android.view.ViewConfiguration.get(ViewConfiguration.java:364)
      at android.view.View.<init>(View.java:3794)
      at android.view.View.<init>(View.java:3898)
      at android.widget.TextView.<init>(TextView.java:677)
      at android.widget.TextView.<init>(TextView.java:671)
      at android.widget.TextView.<init>(TextView.java:667)
      at android.widget.TextView.<init>(TextView.java:663)
      at com.flaredown.flaredownApp.Checkin.tags.TagFragment$3.viewCreation(TagFragment.java:145)
      at com.flaredown.flaredownApp.Checkin.tags.TagFragment$3.viewCreation(TagFragment.java:142)
      at com.flaredown.flaredownApp.Helpers.FlowLayoutHelper.addItem(FlowLayoutHelper.java:44)
      at com.flaredown.flaredownApp.Checkin.tags.TagFragment$2.onSuccess(TagFragment.java:113)
      at com.flaredown.flaredownApp.Checkin.tags.TagFragment$2.onSuccess(TagFragment.java:108)
      at com.flaredown.flaredownApp.Helpers.APIv2.Communicate$27.onResponse(Communicate.java:568)
      at com.flaredown.flaredownApp.Helpers.APIv2.Communicate$27.onResponse(Communicate.java:558)
      at com.flaredown.flaredownApp.Helpers.Volley.JsonObjectExtraRequest$1.onResponse(JsonObjectExtraRequest.java:48)
      at com.flaredown.flaredownApp.Helpers.Volley.JsonObjectExtraRequest$1.onResponse(JsonObjectExtraRequest.java:44)
      at com.android.volley.toolbox.StringRequest.deliverResponse(StringRequest.java:60)
      at com.android.volley.toolbox.StringRequest.deliverResponse(StringRequest.java:30)
      at com.android.volley.ExecutorDelivery$ResponseDeliveryRunnable.run(ExecutorDelivery.java:99)
      at android.os.Handler.handleCallback(Handler.java:739)
      at android.os.Handler.dispatchMessage(Handler.java:95)
      at android.os.Looper.loop(Looper.java:148)
      at android.app.ActivityThread.main(ActivityThread.java:5422)
      at java.lang.reflect.Method.invoke(Native Method)
      at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
      at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

**Comment by thunter1000** [comment]
The same solution can be applied as to issue #42.

Moving the application to the observable model this is where the model updates the view allows previous volley requests is update the current activity instance, or if closed do nothing at all. Also it resumed from the closed state it can restore the entire model, covering handling state across android instance.

**Comment by thunter1000** [comment]
Issue fixed in merge #50
